test/save-in-progress/01-sip-load-fail.e2e.spec.js

```diff
@@ -60,9 +63,10 @@ module.exports = E2eHelpers.createE2eTest(
         value: {},
         status: 401
       }, token)
+      .pause(500)
       .click('.usa-button-primary');
 
-    client.waitForElementVisible('.usa-alert-error', Timeouts.slow);
+    client.waitForElementPresent('.usa-alert-error', Timeouts.slow);
```

Comment:
Interesting. Why wait for it to be present instead of visible?

Reply:
I think this is the same issue as https://github.com/department-of-veterans-affairs/vets-website/pull/7248#discussion_r167942598 where an element can be present but whether Selenium sees it as visible depends on a variety of factors including where it is in relation to the chrome window's viewport.